pkg/client/client_venafi_cloud.go

```diff
@@ -70,9 +70,11 @@ type (
 )
 
 const (
-	vaasProdURL         = "https://api.venafi.cloud"
-	accessTokenEndpoint = "/v1/oauth/token/serviceaccount"
-	requiredGrantType   = "urn:ietf:params:oauth:grant-type:jwt-bearer"
+	// URL for the venafi-cloud backend services
+	VenafiCloudProdURL               = "https://api.venafi.cloud"
+	defaultVenafiCloudUploadEndpoint = "v1/tlspk/uploads"
```

Comment:
Why is this different from the value in [the Venafi Kubernetes Agent Helm chart](https://github.com/jetstack/jetstack-secure/blob/473277b69510d75734a1a714cf09d47583ba53e2/deploy/charts/venafi-kubernetes-agent/templates/configmap.yaml#L17), which looks like:

```yaml
upload_path: "/v1/tlspk/upload/clusterdata"
```

Does it mean that the default value doesn't work as of today?